features/facebookImport/src/model/analyses/utils/analysis-status.js

```diff
@@ -1,14 +1,12 @@
-export function createSuccessStatus(analysisClass) {
+export function createSuccessStatus() {
     return {
-        analysisClass,
         name: "SUCCESS",
         isSuccess: true,
     };
 }
 
-export function createErrorStatus(analysisClass, error) {
+export function createErrorStatus(error) {
```

Comment:
Sorry, but I think this is another place where we should really constrain status with an actual class. I see all kind of things being used as status, with `name` being a constant, but there are also errors, and some other things like "message" and `isSuccess`. Again, not within the scope of this PR, but maybe we could have some issue created, or take a more general approach and do a refinement meeting next week.

Reply:
Yes I agree. There will be another PR to unify statuses between analyses and importers. Right now they use two different mechanism which is far from ideal. After that we can turn the status into a proper object.